src/comap_abi_decoder.rs

```diff
@@ -31,17 +31,21 @@ const HEX_OR_BIGNUM_TRUE: u8    = 0b00010000;
 const IS_ARRAY_MASK: u8         = 0b00001000;
 const IS_ARRAY_FALSE: u8        = 0b00000000;
 const IS_ARRAY_TRUE: u8         = 0b00001000;
+*/
+
+// TODO: Create a more unified library for comap ABI stuff? (Exists both here and in neutron-star currently)
```

Comment:
This can be put into the neutron-common, which is also used for NeutronAddress for instance: https://github.com/earlgreytech/neutron-common

Reply:
Yeah, that was what I was thinking. I actually thought I had written that in the comment, but apparently not.